src/testFixtures/java/uk/gov/di/ipv/cri/common/library/client/TestResourcesClient.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.di.ipv.cri.common.library.client;
+
+import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider;
+import software.amazon.awssdk.http.HttpExecuteRequest;
+import software.amazon.awssdk.http.HttpExecuteResponse;
+import software.amazon.awssdk.http.SdkHttpClient;
+import software.amazon.awssdk.http.SdkHttpFullRequest;
+import software.amazon.awssdk.http.SdkHttpMethod;
+import software.amazon.awssdk.http.auth.aws.signer.AwsV4HttpSigner;
+import software.amazon.awssdk.http.auth.spi.signer.SignedRequest;
+import software.amazon.awssdk.http.crt.AwsCrtHttpClient;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.utils.IoUtils;
+import uk.gov.di.ipv.cri.common.library.aws.CloudFormationHelper;
+import uk.gov.di.ipv.cri.common.library.util.URIBuilder;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+
+public class TestResourcesClient {
+    private final String testHarnessUrl;
+
+    private static final SdkHttpClient CLIENT = AwsCrtHttpClient.builder().build();
+    private static final AwsV4HttpSigner SIGNER = AwsV4HttpSigner.create();
+
+    public TestResourcesClient(ClientConfigurationService clientConfigurationService) {
+        this.testHarnessUrl =
+                CloudFormationHelper.getOutput(
+                        clientConfigurationService.getTestResourcesStackName(),
+                        "TestHarnessExecuteUrl");
+    }
+
+    /** Convert the response body to a string and close its input stream */
```

Comment:
Please remove this

Reply:
This is a documentation (javadoc) comment for the library - we can use those.
This is not a code comment (with `//`), which we're not supposed to use.

Reply:
The reason I asked for this comment to be removed is that the code looks straight forward to me